include/lsst/sphgeom/constants.h

```diff
@@ -33,24 +33,25 @@ namespace sphgeom {
 // Note: given a compiler that does correctly rounded decimal to
 // binary floating point conversions, PI = 0x1.921fb54442d18p1 in
 // IEEE double precision format. This is less than π.
-static double const PI = 3.1415926535897932384626433832795;
-static double const RAD_PER_DEG = 0.0174532925199432957692369076849;
-static double const DEG_PER_RAD = 57.2957795130823208767981548141;
+static constexpr double PI = 3.1415926535897932384626433832795;
```

Comment:
However: I did some more reading and it turns out that (unbeknownst to me) using const or constexpr implies internal linkage. (Even if this weren't the case, an unnamed namespace would be the more modern way to express internal linkage). So you are right to question, and no it isn't useful - I'll remove it. Thanks for catching this!